app/scripts/modules/core/src/managed/versionsHistory/VersionsHistory.tsx

```diff
@@ -156,6 +157,10 @@ const SingleVersion = ({ versionData, pinnedVersions }: ISingleVersionProps) =>
         expandIconType="arrowCross"
         defaultExpanded={versionData.isFocused}
         heading={({ chevron }) => <VersionHeading group={versionData} chevron={chevron} />}
+        onToggle={(isExpanded) => {
+          if (!isExpanded) return;
+          routerState.go(routerState.current, { sha: versionData.key }, { location: 'replace' });
```

Comment:
you can also use a `.` to represent "the current state"
```suggestion
          routerState.go('.', { sha: versionData.key }, { location: 'replace' });
```